inspirehep/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -31,9 +31,11 @@
 {% macro render_record_authors(is_brief, number_of_displayed_authors=10, show_affiliations=true) %}
   {% if record.collaboration %}
     {% set collaboration_displayed = [] %}
-    {% for collaboration in record.collaboration if collaboration != None %}
-      <a href="/search?p=collaboration:'{{ collaboration }}'">{{ collaboration }}</a>
+    {% for collaboration in record.collaboration %}
+      {% if collaboration['value'] %}
```

Comment:
Just tried it in Jinja and there is no error when a key does not exist. (using `collaboration['value']` notation). I guess we are overusing the `.get()` in Jinja templates as it is gentle when keys do not exist.